.github/labeler-pr-triage.yml

```diff
@@ -17,6 +17,8 @@ generators:
   - 'internal/**/*_gen_test.go'
   - 'internal/**/generate.go'
   - 'internal/generate/**/*'
+github_actions:
+  - '.github/workflows/*.yml'
```

Comment:
maybe this should include the yml files in just `.github` dir?

Reply:
I think it's worth adding the yml files in the `.github` dir too, for sure.

Reply:
I took the liberty of add `.github/*.yml` to the list of _github_actions_ patterns.

Reply:
thank you!